pkg/omf/omf.fish

```diff
@@ -119,13 +123,18 @@ function omf -d "Oh My Fish"
 
     case "t" "theme"
       if test (count $argv) -eq 1
-        set -l ostype (uname)
         set -l theme (cat $OMF_CONFIG/theme)
-        set -l regex "[[:<:]]($theme)[[:>:]]"
-        test "$ostype" != "Darwin"; and set regex "\b($theme)\b"
-
-        omf.packages.list --database --theme | column | sed -E "s/$regex/"(omf::em)"\1"(omf::off)"/"
-        omf::off
+        set -l highlight_current "(^|\s)($theme)(\s|\$)"
```

Comment:
But.. but.. `\s` is right there in [re_format]'s man page. :disappointed: 
Can you try now? `[[:space:]]` seems to work for GNU sed and it should work for OS X too.

[re_format]: https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man7/re_format.7.html